test/e2e/tests/tolerate-failure/onboarding/onboarding-new-wallet.spec.ts

```diff
@@ -0,0 +1,115 @@
+import OnboardingMetricsPage from '../../../page-objects/pages/onboarding/onboarding-metrics-page';
+import OnboardingPasswordPage from '../../../page-objects/pages/onboarding/onboarding-password-page';
+
+import SecureWalletPage from '../../../page-objects/pages/onboarding/secure-wallet-page';
+import StartOnboardingPage from '../../../page-objects/pages/onboarding/start-onboarding-page';
+
+import { ALL_POPULAR_NETWORKS } from '../../../../../app/scripts/fixtures/with-networks';
+import FixtureBuilder from '../../../fixture-builder';
+import { getCommonMocks } from '../utils/commonMocks.js';
+import { setupTimerReporting } from '../utils/testSetup.js';
+import Timers from '../../../../timers/Timers.js';
+import AssetListPage from '../../../page-objects/pages/home/asset-list.js';
+
+describe('MetaMask onboarding', function () {
+  // Setup timer reporting for all tests in this describe block
+  setupTimerReporting();
+  it('Creates a new wallet, sets up a secure password, and completes the onboarding process', async function () {
+    await withFixtures(
+      {
+        title: this.test?.fullTitle(),
+        manifestFlags: {
+          testing: {
+            disableSync: true,
+            infuraProjectId: process.env.INFURA_PROJECT_ID,
+          },
+        },
+        useMockingPassThrough: true,
+        disableServerMochaToBackground: true,
+        fixtures: new FixtureBuilder({ onboarding: true })
+          .withEnabledNetworks(ALL_POPULAR_NETWORKS)
+          .build(),
+        testSpecificMock: async (server: Mockttp) => {
+          return [...getCommonMocks(server)];
+        },
+      },
+      async ({ driver }: { driver: Driver }) => {
+        const timer1 = Timers.createTimer(
+          'Time since the user clicks on "Create new wallet" button until "Social sign up" is visible',
+        );
+        const timer2 = Timers.createTimer(
+          'Time since the user clicks on "use SRP" button until "Metamask password" form is visible',
+        );
+        const timer3 = Timers.createTimer(
+          'Time since the user clicks on "Create password" button until "Recovery Phrase" screen is visible',
+        );
+        const timer4 = Timers.createTimer(
+          'Time since the user clicks on "Skip" button until "Metrics" screen is visible',
+        );
+        const timer5 = Timers.createTimer(
+          'Time since the user clicks on "I agree" button until "Onboarding Success" screen is visible',
+        );
+        const timer6 = Timers.createTimer(
+          'Time since the user clicks on "Done" button until "Home" screen is visible',
+        );
+        const timer7 = Timers.createTimer(
+          'Time since the user clicks on "Done" button until "Skip backup" screen and assets list are visible',
+        );
+
+        await driver.navigate();
+        const startOnboardingPage = new StartOnboardingPage(driver);
+        const metrics = await driver.collectMetrics();
+        console.log('Metrics:', metrics);
```

Comment:
this is never stored. Maybe consider adding a `metadata.metrics` object to the JSON output so the timings and browser metrics live together.